.github/workflows/ci.yml

```diff
@@ -54,3 +54,19 @@ jobs:
     # - name: Exit if coverage condition not met
     #   if: ${{ steps.test.outputs.exit_code }} != 0
     #   run: exit ${{ steps.test.outputs.exit_code }}
+
+    - name: Cypress checkout
+      uses: actions/checkout@v2
+
+    - name: Set up Node.js
+      uses: actions/setup-node@v4
+    
+    - name: Install dependencies
+      run: npm i
```

Comment:
Do we need these three steps if they are already executed above? 🤔

Reply:
It is redundant...I will update the PR